### ISSUE TITLE
Docs missing at docs.scipy.org for NumPy >1.16.1 <1.17.0

### ISSUE THREAD

**Issue by mihaic** [numpy/numpy#13811]
There is no documentation at docs.scipy.org for the latest three NumPy versions (1.16.2, 1.16.3, and 1.16.4):

The most recent docs available are for 1.16.1:
https://docs.scipy.org/doc/numpy/
> Welcome! This is the documentation for NumPy 1.16.1, last updated Jan 31, 2019.

Try accessing 1.16.4 directly:
https://docs.scipy.org/doc/numpy-1.16.4/reference/
> The requested URL /doc/numpy-1.16.4/reference/ was not found on this server."

See the list of available versions:
https://docs.scipy.org/doc/

**Comment by rgommers** [comment]
that's because I usually do it, and I didn't get pinged and failed to be proactive. note that there's no interesting delta between 1.16.1 and 1.16.4

I'll see if I can fix this soon.

**Comment by rgommers** [comment]
> I'll see if I can fix this soon.

actually, it will be a while. swamped until scipy. there's no hurry here, no content is missing. also, we have dev docs that are built on every PR merge.

if anyone else with access to the doc server wants to do it, please go for it.

**Comment by rgommers** [comment]
Okay the pdf docs for 1.16.4 don't build due to missing references. I have no time to look into this, and it's low prio. So happy to close or leave open if someone else wants to pick this up.

To build the docs: `cd doc && make dist`.

If you get a `PYTHONPATH` related error, apply this diff:
```
diff --git a/doc/Makefile b/doc/Makefile
index d61d115f0..757312be0 100644
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -1,7 +1,7 @@
 # Makefile for Sphinx documentation
 #
 
-PYVER = 3
+PYVER = 3.6
 PYTHON = python$(PYVER)
 
 # You can set these variables from the command line.
@@ -58,7 +58,7 @@ gitwash-update:
 #
 
 
-INSTALL_DIR = $(CURDIR)/build/inst-dist/
+INSTALL_DIR = $(CURDIR)/build/inst-dist
```

Then the html build works, the pdf build ends with:
```
lyutils' on page 1377 undefined on input line 137144
  Reference `reference/routines.random:module-numpy.random' on page 1377 undefined on input line 137145
  Reference `reference/routines.testing:module-numpy.testing' on page 1377 undefined on input line 137146
Latexmk: Summary of warnings from last run of (pdf)latex:
  Latex failed to resolve 19 reference(s)
  =====Latex reported missing or unavailable character(s).
=====See log file for details.
  Latex failed to resolve 271 citation(s)
Latexmk: Errors, so I did not complete making targets
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs, or warnings treated as errors.
make[2]: *** [Makefile:33: numpy-ref.pdf] Error 12
```

Most of those references are fixed in 1.17.0/master. Could use `-f` for the pdf build to force completion, but then you'd have to check the `numpy-user.pdf` and `numpy-ref.pdf` outputs for issues.

**Comment by mihaic** [comment]
Since, 1.16 is the last series to support Python 2.7, I think it will continue to have users for a while, so it would be helpful to publish the documentation at least for 1.16.4. How about publishing the PDFs with missing references?

**Comment by rgommers** [comment]
the doc build doesn't complete at the moment, `numpy-user.pdf` does not get produced. so I can't publish what's there now. 

> Since, 1.16 is the last series to support Python 2.7, I think it will continue to have users for a while, so it would be helpful to publish the documentation at least for 1.16.4.

I agree that this would be nice to have.

**Comment by charris** [comment]
Might want to wait on 1.16.5 if the documentation is a hassle to produce. I will be making that release fairly soon.

**Comment by mattip** [comment]
NumPy documentation is now located at https://numpy.org/doc. The latest 1.16 documentation is available at https://numpy.org/doc/1.16. As part of the release process, the release manager should [update the docs](https://www.numpy.org/devdocs/dev/releasing.html#build-and-archive-documentation) so this should always reflect the last released version. The docs don't change too much between micro versions (1.16.0 -> 1.16.1 -> 1.16.2 ...) except for bug fixes.

Closing. Please reopen if there is something I did not understand.
